.github/workflows/build.yml

```diff
@@ -46,6 +46,7 @@ jobs:
         if: ${{ github.repository == 'linusg/serenityos-emoji-font' && github.ref == 'refs/heads/main' }}
         run: |
           cp LICENSE build/
+          cp SerenityOS-Emoji.css build/
```

Comment:
Let's copy it into `build` _and_ `pages` so that the live website can use it as well :)

Needs an update to the HTML in `main.py` to the replace the font CSS there.